boot/zephyr/CMakeLists.txt

```diff
@@ -161,6 +155,12 @@ if(CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256 OR CONFIG_BOOT_ENCRYPT_EC256)
       ${MBEDTLS_ASN1_DIR}/src/asn1parse.c
       ${MBEDTLS_ASN1_DIR}/src/platform_util.c
       )
+  # Since here we are not using Zephyr's mbedTLS but rather our own, we need
+  # to set MBEDTLS_CONFIG_FILE ourselves. When using Zephyr's copy, this
+  # variable is set by its Kconfig in the Zephyr codebase.
+  zephyr_library_compile_definitions(
+    MBEDTLS_CONFIG_FILE="${CMAKE_CURRENT_LIST_DIR}/include/mcuboot-mbedtls-cfg.h"
```

Comment:
CONFIG_MBEDTLS_CFG_FILE holds the this cfg.h file name, can we utilize this config here??

Reply:
we are utilizing the same

Reply:
use value of CONFIG_MBEDTLS_CFG_FILE instead of mcuboot-mbedtls-cfg.h file name then it is configurable through CONFIG